spinalcordtoolbox/image.py

```diff
@@ -16,7 +16,8 @@
 import warnings
 import logging
 import math
-from typing import Sequence
+from typing import Sequence, Tuple
+from copy import deepcopy
```

Comment:
With this change, there are now redundant imports of `deepcopy` that give me some pause:

https://github.com/spinalcordtoolbox/spinalcordtoolbox/blob/9157b5956aacd26a8a947be84cf0cd3f7165f449/spinalcordtoolbox/image.py#L359-L364

I tried to understand why `Image` delays the import of `deepcopy` until its methods are run, rather than at the top of the file, but these lines were added 8 years ago (https://github.com/spinalcordtoolbox/spinalcordtoolbox/commit/a5ba8b2bfba902403ae6b4d9f7c58ef35359add1), and so I can't find any discussion about the choice.

Still, it might be worth removing these redundant imports, now?

Reply:
That's right, you should remove the redundant import.

I think there are two typical reasons for having a non-toplevel import (that is, inside a function):
* To avoid problems with circular imports.
* To avoid loading an expensive module if it's not needed.

I don't think either of these applies to the `copy` module from the standard library.